Twig/Extension/SecurityExtension.php

```diff
@@ -0,0 +1,49 @@
+<?php
+
+/*
+ * This file is part of the Sonata Project package.
+ *
+ * (c) Thomas Rabaix <thomas.rabaix@sonata-project.org>
+ *
+ * For the full copyright and license information, please view the LICENSE
+ * file that was distributed with this source code.
+ */
+
+namespace Sonata\AdminBundle\Twig\Extension;
+
+use Sonata\AdminBundle\Admin\AdminInterface;
+
+final class SecurityExtension extends \Twig_Extension
+{
+    /**
+     * @var AdminInterface
+     */
+    private $admin;
+
+    /**
+     * @param AdminInterface $admin
+     */
+    public function setAdmin(AdminInterface $admin)
```

Comment:
This looks so wrong. Why would you inject one `AdminInterface` instance here? Twig extensions are stateless